docs/tools/vega-wallet/CLI-wallet/create-wallet.md

```diff
@@ -102,7 +102,13 @@ vegawallet init
 
 ### Import existing wallet
 
-Use your mnemonic (recovery phrase) to import a wallet you created in the past into a new version of the wallet software. If you were not provided with a mnemonic (recovery phrase) or have lost it, you will need to create a new wallet/keypair. Use the command below to import the plain-text file with your mnemonic in it.  
+Use your mnemonic (recovery phrase) to import a wallet you created in the past into a new version of the wallet software. If you were not provided with a mnemonic (recovery phrase) or have lost it, you will need to create a new wallet/keypair. Use the command below to import the plain-text file with your mnemonic in it. 
```

Comment:
> you will need to create a new wallet/keypair 

this slash sounds like an amalgam, a shortcut or make things look like they can be quoted interchangeably, which is not accurate.

> you will need to create a new wallet and generate new keys

I think that would be more accurate.